examples/BridgeSerial/BridgeSerial.ino

```diff
@@ -0,0 +1,46 @@
+/*
+
```

Comment:
First Introduce the BridgeSerial objective. I understand this is:
Allows to relay UART data to/from a PC terminal to/from LORA shield.

Reply:
done